dump1090.c

```diff
@@ -922,9 +928,13 @@ int main(int argc, char **argv) {
     // Set sane defaults
     modesInitConfig();
 
+#ifdef WIN32
```

Comment:
@mutability I would propose replacing both windows and posix section with function call:
initSignals(&Mode.exit);